pyproject.toml

```diff
@@ -12,13 +12,13 @@ license="MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.9,<3.11"
 tensorboard = "^2.10.0"
 wandb = "^0.13.11"
 gym = "0.23.1"
 torch = ">=1.12.1"
-stable-baselines3 = "2.0.0"
-gymnasium = ">=0.28.1"
+stable-baselines3 = "^2.4.0"
```

Comment:
I suspect we will have a new release of sb3 with support for gymnasium v1.1.0 as no changes seem to be required on their end (https://github.com/DLR-RM/stable-baselines3/pull/2095)